pkgs/by-name/cz/czkawka/package.nix

```diff
@@ -85,11 +91,14 @@ let
       install -Dm444 -t $out/share/metainfo data/com.github.qarmin.czkawka.metainfo.xml
     '';
 
+    nativeInstallCheckInputs = [
+      versionCheckHook
```

Comment:
Why?

Reply:
This has the advantage of being run at build time and thus is less prone to being forgotten.
It is recommended over `passthru.tests.version` for those reasons by the manual.

Reply:
Hum...
The manual does not explain why the `versionCheckHook` should be preferred over the `passthru.tests.version`, it just states it in a mid-note without elaborating further.

"Prone to being forgotten" can be a more or less compelling argument for adding the check hook, but not for removing the passthru test.
Even because, specifically for Nixpkgs, ofBorg runs the passthru tests.

Further, the hook runs at "build time" while the passthru test is a derivation.

Let's keep both.